recipe/meta.yaml

```diff
@@ -21,7 +21,7 @@ requirements:
   run:
     - python
     - setuptools
-    - enum34  # [not (py35 or py34)]
+    - enum34  # [py<34]
```

Comment:
Does this work ok with re-rendering? Had some issues with selectors like this in the past. Putting `py2k` might work better if it does have issues.